transform/binary-plugin/binary_plugin.go

```diff
@@ -0,0 +1,78 @@
+package binary_plugin
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+
+	"github.com/konveyor/crane-lib/transform"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type BinaryPlugin struct {
+	commandRunner
+	log logrus.FieldLogger
+}
+
+func NewBinaryPlugin(path string) transform.Plugin {
+	return &BinaryPlugin{commandRunner: &binaryRunner{path: path}, log: logrus.New().WithField("path", path)}
```

Comment:
nit: follow up but the path is probably not the best name "plugin" might be better?